contracts/cw20-atomic-swap/src/contract.rs

```diff
@@ -37,7 +36,7 @@ pub fn handle<S: Storage, A: Api, Q: Querier>(
     match msg {
         HandleMsg::Create(msg) => {
             let sent_funds = env.message.sent_funds.clone();
-            try_create(deps, env, msg, Balance::Native(NativeBalance(sent_funds)))
+            try_create(deps, env, msg, Balance::from(sent_funds))
```

Comment:
Nitpick:

`Balance::from(sent_funds)` == `sent_fund.into()` right?

I prefer the second, even if it is a bit more magic. I will merge without, just a note for the future.

Reply:
Sure, will take those into account; along with implementing and using the `From` trait from the beginning.